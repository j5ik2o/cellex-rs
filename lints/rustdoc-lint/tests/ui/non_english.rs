#![allow(dead_code)]

/// 日本語の説明です。
pub struct Foo;

fn main() {}
