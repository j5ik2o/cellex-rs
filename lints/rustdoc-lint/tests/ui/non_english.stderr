warning: rustdoc は英語で記述してください
  --> $DIR/non_english.rs:3:5
   |
LL | /// 日本語の説明です。
   |     ^^
   |
   = help: 日本語などの全角文字はドキュメントコメントでは使わず、別資料に移動してください
   = note: AI向けアドバイス: ドキュメント生成時には英語の説明文と `rust` のコード例を組み合わせて出力してください。
   = note: `#[warn(rustdoc_lint)]` on by default

warning: 1 warning emitted

